torch/distributed/fsdp/_runtime_utils.py

```diff
@@ -53,6 +53,36 @@ class _PrefetchMode(Enum):
     FORWARD = auto()
 
 
+def fsdp_state_has_default_pg(state: _FSDPState) -> bool:
+    """Indicates whether FlatParamHandle has the default process group.
+
+    Args:
+        handle (_FSDPState): FSDP State object
+
+    Returns:
+        bool: True if the ProcessGroup of the _FSDPState object is the default process group. False
+            otherwise.
+    """
+    from torch.distributed.distributed_c10d import get_process_group_ranks
+    if state.process_group is None:
+        # If no process group is attached to the _FSDPState, assume it uses default process group.
+        return True
+    return len(get_process_group_ranks(
+        state.process_group)) == dist.get_world_size()
+
+
+def fsdp_state_pg_ranks(state: _FSDPState) -> Tuple[int]:
```

Comment:
```suggestion
def fsdp_state_pg_ranks(state: _FSDPState) -> Tuple[int, ...]:
```
is probably what you actually mean.